duckduckgo/iOS
# text updates

Reviewer: Caine or Mia
Asana:

* https://app.asana.com/0/414235014887631/521587462118443
* https://app.asana.com/0/414235014887631/552319851838328

CC:

**Description**:

Updates truncated text on smaller screens
Updates the mixed encryption state message

**Steps to test this PR**:

1. Check the privacy dashboard screens on devices of various sizes and check for consistency ensuring text is not truncated
1. Visit a secure site and check the encryption state message is correct
1. Visit a mixed encryption site (e.g. https://english.sogou.com/ ) and check the mixed encryption state text is correct
1. Visit a site with no encryption and check the encryption state message is correct
